relay-server/src/actors/events.rs

```diff
@@ -754,6 +764,31 @@ impl EventProcessor {
         }));
     }
 
+    /// Extracts the timestamp from the minidump and uses it as the event timestamp.
+    #[cfg(feature = "processing")]
+    fn write_minidump_timestamp(event: &mut Event, minidump_item: &envelope::Item) {
+        use std::time::SystemTime;
+
+        assert_eq!(
```

Comment:
do we have such asserts already? This is clearly a programmer error, so using panic in this case seems okay, or maybe use the typestate pattern to make this impossible to call?

Reply:
our existing pattern is to use dbg_assert

Reply:
Definitely do not assert and rather return an error (or swallow it silently). Given how the other functions here are coded (e.g. `parse_msgpack_breadcrumbs`), I would just early return.